Use NPM 6.x on Node 10.x

sandermarechal opened the pull request:
Use npm 6.x instead of npm 8.0 when using node 10.x, fixes #294

mistraloz commented:
Thx for your PR but please follow instruction at top of each files : 
`# DO NOT EDIT THIS FILE : Make yours changes in /utils/Dockerfile.*.blueprint)`

To apply this change only for node10, please do like that : 
`{{ if eq "10" $node_version }}@^6.14{{end}}`

It's require [orbit](https://github.com/gulien/orbit/) on your computer.

sandermarechal commented:
I have updated the PR and made the change in the blueprint file. After running orbit it make the correct changes to the Dockerfiles, but orbit also removed all the `DO NOT EDIT THIS FILE` lines. Is that supposed to happen? I have not committed the removal of these lines, just the changes to `npm install`.

mistraloz commented:
Thx for this fix @sandermarechal . The related lines at the top of files are added by sed after obit generate (it's on the Makefiles script). But as you did it's perfect. After checks passed, i will merge it. 

PS : if you know another better tool/process to manage template, i take all suggestions.

sandermarechal commented:
Thank you. I don't know a better way to manage templates. At least orbit is easy to install, as opposed to node+mustache or something.
